From 8fe5c9421acfaff35b637e7ad55d1df598bb7081 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 22 Dec 2020 09:22:10 +0800
Subject: [PATCH] unix-Makefile.tmpl: don't add prefix for libdir

we had pass libdir to Configure, don't use prefix again to
avoid engineer dir set to:
/opt/poky/3.2+snapshot/sysroots/x86_64-w64-mingw32/usr/opt/poky/3.2+snapshot/sysroots/x86_64-w64-mingw32/usr/lib/engines-1_1

Upstream-Status: Inappropriate[oe-specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 Configurations/unix-Makefile.tmpl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Configurations/unix-Makefile.tmpl b/Configurations/unix-Makefile.tmpl
index bbafb98..eecb63e 100644
--- a/Configurations/unix-Makefile.tmpl
+++ b/Configurations/unix-Makefile.tmpl
@@ -244,7 +244,7 @@ LIBDIR={- our $libdir = $config{libdir} || "lib";
           File::Spec::Win32->file_name_is_absolute($libdir) ? "" : $libdir -}
 ENGINESDIR_dev={- use File::Spec::Win32;
                   our $enginesdir =
-                      File::Spec::Win32->catdir($prefix,$libdir,
+                      File::Spec::Win32->catdir($libdir,
                                                 "engines-$sover_dirname");
                   our ($enginesdir_dev, $enginesdir_dir, $enginesdir_file) =
                       File::Spec::Win32->splitpath($enginesdir, 1);
-- 
2.17.1

